src/test/java/synapseawsconsolelogin/AuthTest.java

```diff
@@ -267,4 +267,11 @@ public void testRedirectURIsDefault() {
 		assertEquals(Collections.singletonList("baz"), auth.getRedirectURIs("baz"));
 	}
 
+	@Test
+	public void testSanitizeValues() {
+		assertEquals("Sage-Bionetworks  A Better Science Company ", Auth.sanitizeTagValue("Sage-Bionetworks (A Better Science Company)"));
+		assertEquals("abc XYZ 123  _.:/=+ -@", Auth.sanitizeTagValue("abc XYZ 123 \t_.:/=+\\-@"));
+		assertEquals("                 ", Auth.sanitizeTagValue("!#$%^&*(){}|\"';<>"));
```

Comment:
should commas be sanitized as well?

Reply:
They are, but not included in the test.  I will update the test.